README.md

```diff
@@ -23,7 +23,7 @@ the contents of this repository.
 Running the automation script below to setup everything:
 
 ```console
-curl -fsSL "https://tinyurl.com/setup-dots" | bash -c
+bash -c "$(curl -fsSL 'https://tinyurl.com/setup-dots')"
```

Comment:
Instead of using the `-c` flag, using the `-s` flag makes more sense since the `curl` command can then be piped into the `bash -s` along with whatever positional arguments the script might accept in the future. I plan on adding some optional arguments to pass to the script like `--dry-run` and more, so piping the output of the `curl` command to `bash -s` will be more helpful in this regards.

See the "OPTIONS" section in `man bash` for more information about the aforementioned flags.